src/css/ContactLinks.scss

```diff
@@ -36,11 +39,11 @@ $copyWriteTextColor:#969696;
 }
 
 /* // Medium devices (tablets, 768px and up) */
-@media (min-width: 768px) { 
+@media (min-width:768px) { 
   .contactLinks{
   
```

Comment:
remove extra line here

Reply:
Where are you saying the extra line is?

Reply:
Not sure this comment is valid. For some reason I'm making comments in specific places, but Github changes where I made the comment.